### DISCUSSION TITLE
Add the list of projects?

### DISCUSSION THREAD

**Discussion by iacore** [david-vanderson/dvui#49] [General]
@david-vanderson Should this be in readme or Github Wiki?

**Comment by david-vanderson** [comment]
Readme - let's try it right at the top replacing the "Status:" line with something like "Projects using DVUI include: list".  Great idea!

**Comment by david-vanderson** [comment]
How does that look?  Is the name/link for your project correct?

**Reply by iacore** [reply]
it's correct!

**Comment by iacore** [comment]
@david-vanderson Why is the demo project listed? It is technically not a "project" and now listed twice in readme.

**Comment by david-vanderson** [comment]
Yeah I thought it looked good in the list (and easy to see with a quick scan) and removed the other, but then there wasn't any text saying what it was about.  I think it's fine to have it twice - once for people quickly scanning, and once for people carefully reading.  I don't feel strongly about it though.

**Reply by iacore** [reply]
it's confusing to me: demo mixed with other projects

demo is documentation, while other projects is showcase

**Reply by david-vanderson** [reply]
It's a good point, but I want to highlight the demo repo so anyone doing a quick read knows about it.  I'm worried that they'll try to start from one of the other projects (that's what I would do).  I've added an explanation for it and removed the other reference.  Let's try that for a while and see if anyone gets confused.

**Reply by iacore** [reply]
well you should put the project list lower than the "how to use as library", and link to the demo where the list is now. For our case it should be at the end of readme.

for new users they probably don't need to know about the projects that use DVUI. that's why I propose putting it at the end.

you can see [imgui's readme](https://github.com/ocornut/imgui/) where "Gallery" is very down

**Reply by david-vanderson** [reply]
I mostly agree.  I've moved the dvui-demo link to the section for how to use as a library, and put the list of projects just below the screenshot.  I think that makes sense since the screenshot and project list are both there to give the reader confidence that this is useful.
